godotproject/Player.gd

```diff
@@ -435,20 +435,19 @@ func _physics_process(delta):
 			# with small upwards boost
 			stop_swing()
 			
-			# A unit vector in the direction of the mouth to the tip of tongue.
-			# It will be used to calculate the launch direction.
-			# The direction is calculated as the vector perpendicular to the current velocity,
-			# we divide by the magnitude to make it a unit vecotr. This helps to normalize it.
-			var tongue_direction = Vector2(velocity.y, -1*velocity.x).normalized()
+			var frog_pos = get_global_transform().xform(Vector2.ZERO)
+			var swing_vec = swing_pivot_position - frog_pos 
+			var swing_vec_norm = swing_vec.normalized()
```

Comment:
Could you possibly rename this to `tongue_vec`? I don't want it to be confused with the `swing_vec` used earlier on in the file, which is the opposite vector (from swing position to frog)

Reply:
sure